admin/scss/_actionTabs.scss

```diff
@@ -125,7 +125,7 @@ $border: 1px solid darken(#D9D9D9, 15%);
 					font-size: 12px;
 				}	
 
-				#Form_AddForm_PageType_Holder ul {
+				#Form_AddForm_PageType ul {
```

Comment:
I'm unsure this even applies to anything? The feature this applies to was never implemented, but got merged with the bottom actiontabs in 2012. :-/